casemanagementv1/case_management_v1_integration_test.go

```diff
@@ -0,0 +1,380 @@
+// +build integration
+
+/**
+ * (C) Copyright IBM Corp. 2020.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package casemanagementv1_test
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"io/ioutil"
+	"strings"
+	"encoding/json"
+
+	"github.com/joho/godotenv"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/IBM/go-sdk-core/v4/core"
+	"github.com/IBM/platform-services-go-sdk/casemanagementv1"
+)
+
+const externalConfigFile = "../case_management.env"
+
+// Config
+var (
+	service           *casemanagementv1.CaseManagementV1
+	err               error
+	config            map[string]string
+	configLoaded      bool = false
+	testAccountID     string
+	testAccountAPIKey string
+)
+
+// Test values
+var offeringType, _ = service.NewOfferingType(casemanagementv1.OfferingType_Group_CrnServiceName, "cloud-object-storage")
+
+var (
+	caseNumber         = "CS1310378"
```

Comment:
@huydoan2, I think having a fixed case number makes the integration testing fragile. Tests will fail if this case is permanently closed or customer provided their own credentials for integration testing. Instead of this, I did the following in other repos.

1. Run a test that creates a new case
2. Store the newly create case number (maybe a static variable?)
3. Reuse the new case number in subsequent tests

Reply:
I remember doing exactly what you said. Not sure why that hard-coded case number is still there. Will fix!

Reply:
Also, if there are static values that the test needs to use (perhaps an id of a resource that the test depends on but doesn't create itself) we can easily add these values as additional config properties to the case_management.env file and then the test could just retrieve that during its initialization.

Reply:
We are good. Just fixed the usage of static variable. Thanks, Phil.